m4_sessions/1708-inning/group_five.md

```diff
@@ -2,20 +2,30 @@
 
 ### Session Title: The Robots are Coming: The future of AI
 
+### Description
+The robots are coming! Come to our session for an intro to computer vision and discuss the larger societal implications of artificial intelligence (and how they’ll take our jerbs!). And also we are going to watch cartoons.
+
 ### Session Outcomes: 
 - What does it mean to me as a programmer
 - Want people to consider how AI benefits or threatens their lives as devs
 - What AI is currently available, near future, and distant future
 - Understanding of what AI is (definitions and relevant vocabulary)
 
 ### Session Outline:
-- 10 to 15 min Intro (Icebreaker)
-- 20 min content video
-- Discuss the video
-- Break out in to small groups and discuss 
-  - AI in your lives and how it could affect you our those you know
-  - What kind of AI have you seen and what might you like to create or see create
-- Closing discussion on what the groups talked about and final thoughts from mod 4
+- 15 minutes: Introductions
+  - Tell us your name
+  - Your Cohort
+  - Something you think is cool that uses Artificial Intelligence or something you hope someone will make with Artificial Intelligence
 
+- 5 minutes: Introduction to AI with Computer Vision
+- Discuss the video
+- 12 minutes: Kurzgesagt on Automation
+- 25 minutes: Small group discussions
+  - Ideas to consider:
+    - Should there be AI regulation? If so, who should be doing the regulating?
+    - How could (or does) AI make society better off?
+    - How could (or does) AI make society worse off?
+    - What rad futuristic technologies do you think we will create in our lifetimes using AI?
+- 15 minutes: Bring it Back Together - Large Group Discussion
```

Comment:
Plan to have some large group closing.  Do you want everyone to share ideas?  Do you want a new whole group question?  Do you want people to have a final idea or take away that they share with the group?